app/controllers/v1/pacbio/plates_controller.rb

```diff
@@ -21,8 +21,8 @@ def create
           render json: body, status: :created
         else
           # JSON-API formatted errors.
-          errors = @plate_creator.errors.map { |title, detail| { title: title, detail: detail } }
-          render json: { errors: errors },
+          errors = @plate_creator.errors.map { |title, detail| { title:, detail: } }
+          render json: { errors: },
```

Comment:
Is this new syntax?, like in Js with: ```{ errors }```

Reply:
Yeah, it's a Ruby 3.1 thing. Rubocop is already suggesting the new syntax as soon as it detects Ruby 3.1.

If I remember correctly, the main reason they didn't match the JS syntax for this was because it can interfere with the way ruby can distinguish between blocks and hashes.